src/main/cpp/t08_fibb.cpp

```diff
@@ -21,5 +21,21 @@
 using namespace std;
 
 int t08_fibb() {
-
+int n, a = 0, b = 1, c=0;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [The scope of the variable 'c' can be reduced.](https://app.codacy.com/manual/thejerome/IFMO_CPP_programming_20190915/pullRequest?prid=4592457)